Up_Squared/Ubuntu/2_Clone_Repo_Disable_console_sethost.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+if [ $(id -u) -ne 0 ]; then
+   echo >&2 "Must be run as root"
+   exit 1
+fi
+
+set -e
+set -x
+
+. config.env
+
+tput setaf 3
+echo "Cloning Companion Repo"
+tput sgr0
+
+pushd /home/$NORMAL_USER/GitHub/companion/Up_Squared/Ubuntu
+
+tput setaf 3
+echo "Running Scripts"
+tput sgr0
+
+tput setaf 3
+echo "Setting Hostname to apsync"
+tput sgr0
+./set_hostname.sh   # reset the machine's hostname
+
+tput setaf 3
+echo "Removing unused packages"
+tput sgr0
+apt autoremove -y # avoid repeated no-longer-required annoyance
+
+apt-get remove -y unattended-upgrades
+
+tput setaf 3
+echo "Setting up rc.local"
+tput sgr0
+./ensure_rc_local.sh
+
+# tput setaf 3
```

Comment:
Are these commented out lines required?  never used?  sometimes used?

Reply:
It is not required, I think I can remove them if need to.